README.md

```diff
@@ -54,6 +55,37 @@ quoting mechanism.
 
 After the conversion, the bytes can be use in the field `metadata` when originating the contract.
 
+## Generating the contract storage
```

Comment:
I think that this section should be removed now.

The new info about metadata generation should be in the section just above and the ones about storage should be in the dedicated file (see previous comment).

Reply:
I've removed this and added some info in compilation.md.